### PR TITLE
WIP: (SIMP-6600) Update 6.4 Doc

### PR DESCRIPTION
**Pull request by jeannegreulich**
- add an example for creating a new environment

SIMP-6600 #close

### CODE REVIEW

**Comment by lnemsick-simp** [comment]
Working on minor tweaks.  Please do not merge yet.

**Comment by op-ct** [comment]
Working on review and tweaks—Please do not merge yet.

File: docs/user_guide/Upgrade_SIMP/General_Upgrade_Instructions.rst [hunk 1]
```diff
@@ -63,131 +78,51 @@ Incrementally upgrading systems using local repositories
         .. code-block:: sh
 
            # Unpack the new SIMP ISO's RPMs into yum repositories
-           unpack_dvd </path/to/ISO>
+           $ unpack_dvd </path/to/ISO>
```
[the remark this answers is not in the document]
Comment (lnemsick-simp) [reply docs/user_guide/Upgrade_SIMP/General_Upgrade_Instructions.rst]: # does not render properly.  It shows up as a greyed out comment.

**Comment by jeannegreulich** [comment]
Working on the changes recommended by Chris.

**Comment by jeannegreulich** [comment]
This was updated and merged with PR https://github.com/simp/simp-doc/pull/347.  (Sorry I thought it had been closed but someone had just changed it to WIP.)

**Pull request closed without merging**